tests/ha/multi_pod_failure/test_multi_data_server_pod_failure.py

```diff
@@ -0,0 +1,635 @@
+
+    def teardown_method(self):
+        """
+        This function will be invoked after each test function in the module.
+        """
+        LOGGER.info("STARTED: Teardown Operations.")
+        if self.s3_clean:
+            LOGGER.info("Cleanup: Cleaning created IAM users and buckets.")
+            resp = self.ha_obj.delete_s3_acc_buckets_objects(self.s3_clean)
+            assert_utils.assert_true(resp[0], resp[1])
+        if self.restore_pod:
+            for pod_name in self.pod_name_list:
+                if len(self.pod_dict.get(pod_name)) == 2:
+                    deployment_name = self.pod_dict.get(pod_name)[1]
+                    deployment_backup = None
+                else:
+                    deployment_name = self.pod_dict.get(pod_name)[2]
+                    deployment_backup = self.pod_dict.get(pod_name)[1]
+                resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                               restore_method=self.restore_method,
+                                               restore_params={"deployment_name": deployment_name,
+                                                               "deployment_backup":
+                                                                   deployment_backup})
+                LOGGER.debug("Response: %s", resp)
+                assert_utils.assert_true(resp[0], "Failed to restore pod by "
+                                                  f"{self.restore_method} way")
+                LOGGER.info("Successfully restored pod %s by %s way",
+                            pod_name, self.restore_method)
+        if self.restore_node:
+            for node_name in self.node_name_list:
+                LOGGER.info("Cleanup: Power on the %s down node.", node_name)
+                resp = self.ha_obj.host_power_on(host=node_name)
+                assert_utils.assert_true(resp, f"Host {node_name} is not powered on")
+                LOGGER.info("Cleanup: %s is Power on. Sleep for %s sec for pods to join back the"
+                            " node", node_name, HA_CFG["common_params"]["pod_joinback_time"])
+                time.sleep(HA_CFG["common_params"]["pod_joinback_time"])
+        if self.restore_ip:
+            for node_ip in self.node_ip_list:
+                LOGGER.info("Cleanup: Get the network interface up for %s ip", node_ip)
+                node_iface = self.ip_dict.get(node_ip)[0]
+                worker_obj = self.ip_dict.get(node_ip)[1]
+                worker_obj.execute_cmd(cmd=cmd.IP_LINK_CMD.format(node_iface, "up"),
+                                       read_lines=True)
+                resp = sysutils.check_ping(host=node_ip)
+                assert_utils.assert_true(resp, "Interface is still not up.")
+        if os.path.exists(self.test_dir_path):
+            sysutils.remove_dirs(self.test_dir_path)
+        # TODO: As cluster restart is not supported until F22A, Need to redeploy cluster after
```

Comment:
we may need to chnage this message as:
Will need DTM support for pod restart and recovery so need to redeploy